### PR TITLE
fix: workflow shows no logs

### PR DESCRIPTION
**Pull request by alexcodelf**
**Problem:**

- Workflow step shows no logs and return error.
- Resource run list stream doest not return record after succeeded. https://github.com/seal-io/walrus/issues/2192#issuecomment-1996902795

**Solution:**

- As the resource run has two stage, the workflow step needs update to fetch two step logs.
- Return plan and apply records.

**Related Issue:**
#2249

### CODE REVIEW

File: pkg/workflow/step/service_step.go [hunk 1]
```diff
@@ -31,69 +72,53 @@ commonPath="$serverURL/v1/projects/$projectID/environments/$environmentID"
...
-	timeout=$((timeout - factor))
-done
+# Watch run plan logs until the plan finished.
+common_get "$commonPath/resources/$resourceName/runs/$runID/log?jobType=plan&watch=true" --compressed
+
+# Check run status, wait until run status is running.
+wait_run_status "$resourceName" "$runID" "Running"
+
+# Watch run logs until the run finished.
+common_get "$commonPath/resources/$resourceName/runs/$runID/log?jobType=apply&watch=true" --compressed
```
Comment (gitlawr) [inline pkg/workflow/step/service_step.go:118]: Is the compressed arg valid? Looks like common_get is not handling it.

Comment (alexcodelf) [reply pkg/workflow/step/service_step.go]: updated.

**Review by gitlawr — APPROVED** [review]
LGTM

**Review by aiwantaozi — APPROVED** [review]
LGTM

**Pull request merged**